### ISSUE TITLE
Empty string error with lz4.frame

### ISSUE THREAD

**Issue by sasaabuse** [python-lz4/python-lz4#46]
lz4.frame fails with an empty string (py27, macOS, lz4 0.10.1)

```
In [32]: import lz4.frame

In [33]: lz4.frame.decompress(lz4.frame.compress(b''))
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-33-e1bd89fc259b> in <module>()
----> 1 lz4.frame.decompress(lz4.frame.compress(b''))

RuntimeError: LZ4F_getFrameInfo failed with code: ERROR_frameType_unknown
```

**Comment by jonathanunderwood** [comment]
Thanks - am reworking the frame stuff, so will be sure to catch this case. I am on the fence as to whether this particular case should actually raise some sort of exception though - I'd argue that an empty string is invalid input. What were you expecting to happen in this case?

**Comment by sasaabuse** [comment]
I'm not familiar with lz4 algorithm nor the frame format, so I don't know how empty string should be handled. My use case is to use lz4 as a drop-in replacement for python stdlib zlib (which works with empty strings as well), and one of my test cases caught this.

However, `lz4.block` seems to work perfectly for me.

**Comment by jonathanunderwood** [comment]
Should be fixed in 0.12.0. Please re-open if not, with a reproducer if possible.
